projects/js-packages/react-data-sync-client/src/DataSyncHooks.ts

```diff
@@ -63,7 +70,7 @@ export function useDataSync<
 	params: Record< string, string | number > = {}
 ): DataSyncHook< Schema, Value > {
 	const datasync = new DataSync( namespace, key, schema );
-	const queryKey = [ key, ...Object.values( params ) ];
+	const queryKey = [ key, ...Object.values( params ).sort() ];
```

Comment:
🤔 I'm not 100% sure this is right. I think the key order matters. Imagine this scenario: 

You have two parameters: `foo` and `bar`. If you're sorting by value then `{foo:1,bar:2}` will have the same query cache key as `{foo:2,bar:1}`.

I think we'd want to sort by key for consistency, then pull out the values. Something like: 

```
const queryKey = [ key, ...Object.entries.sort( ( [ a ], [ b ] ) => b-a ).map( [ k, v ] => v ) ];
```

Reply:
A-ha. Yes, nice catch. Fully agree.

Reply:
Fixed in 39610c3.